src/pages/Header.js

```diff
@@ -0,0 +1,40 @@
+import { NavLink } from 'react-router-dom';
+import React from 'react';
+
+const Header = () => (
+  <header>
+    <NavLink to="/">
+      <h1 className="title">Space Travelers Hub</h1>
+    </NavLink>
+    <nav className="header__nav">
+      <ul>
+        <li>
+          <NavLink
+            className={({ isActive }) => (isActive ? 'active' : '')}
+            to="/rockets"
+          >
```

Comment:
The directory '/rockets' is empty. I suggest you change it to '/' since that is the homepage. Thank you